frontends/web/src/components/language/language.jsx

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-import { Component } from 'preact';
+import { Component, h } from 'preact';
 import { translate } from 'react-i18next';
 // import { Button } from '../forms';
 // import Dialog from '../dialog/dialog';
-// import style from './language.css';
+// import * as style from './language.css';
```

Comment:
Remove these as well?

Reply:
I mean, it doesn't feel like a `Dialog` should have anything to do with the language anyway, in the language module that is.

Reply:
What do you mean? I likely changed this with a global search and replace. The file describes the language switcher (and not a more general "language module"), which we only deactivated because we didn't want to keep the German translation up-to-date for now. In any case, this was again not the focus of my PR. 🙂

Reply:
Sorry, that was out of scope indeed. Good that you ignored it :)